main.cpp

```diff
@@ -59,9 +59,9 @@ int main()
   file_directory_results << "stereoboard_database/database_stereoboard_" << number_stereoboard << "/" <<
                          configuration_board << "/take" << number_take << "/result_stereo.csv";
 
-
+  struct cam_state_t cam_state;
   //initialize for edgeflow
-  edgeflow_init(128, 94, 0);
+  edgeflow_init(128, 94, 0,&cam_state);
```

Comment:
as the comments describe in the init, if your not using the state you should pass a NULL pointer.